.circleci/readme.md

```diff
@@ -149,44 +154,37 @@ default medium config and Large containers.
 ## Updating the config
 For configuration changes meant to be permanent in the Apache repo you should never edit
 the `config.yml` file manually. Instead, you should edit the `config-2_1.yml` file and then
```

Comment:
So to my earlier question, if we could name this something like `config-template.yml` it'd make the relationship between things a lot more innately clear I think.

Reply:
+1 to renaming the file, even if it's only because I never remember where to put the hyphen and where the underscore in `config-2_1.yml`.

Regarding the new name for the file, I'm not sure if calling it a template is clear enough, and we also refer to the FREE/PAID configs as templates. Maybe it's the main config file, or the manually-generated file? In CircleCI terminology, `config.yml` is the "expanded config" and `config-2_1.yml` is the "reusable config". For us, `config-2_1.yml` is the file that we manually edit, and `config.yml` is the one that we automatically generate.

Reply:
> even if it's only because I never remember where to put the hyphen and where the underscore in config-2_1.yml.
This. 1000x this.

Don't have any ideas beyond referring to it as "template". Could also refer to all three with .template in them or something?

Reply:
I cannot think on a better name either, and "template" is definitively better that "2_1", so I'm renaming it.